zerver/models.py

```diff
@@ -152,6 +152,7 @@ class Realm(ModelReprMixin, models.Model):
     date_created = models.DateTimeField(default=timezone.now) # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True) # type: Optional[Stream]
     deactivated = models.BooleanField(default=False) # type: bool
+    default_language = models.CharField(default=u'en', max_length=50) # type: text_type
```

Comment:
Yes, exactly, that should be part of this pull request (but put it at the start of the branch so we can easily merge that commit before the rest of the PR).